postgresql/_dumps/structs.sql

```diff
@@ -41,10 +41,10 @@ CREATE TABLE data_types (
   _float64 numeric(10,6),
   _bool boolean,
   _string text,
-  _date timestamp without time zone,
```

Comment:
yes of course, see the line right below it... `_ptrdate timestamp without time zone`  .. there are test cases with and without the time zone